source/extensions/filters/http/dynamic_forward_proxy/proxy_filter.cc

```diff
@@ -121,11 +121,14 @@ Http::FilterHeadersStatus ProxyFilter::decodeHeaders(Http::RequestHeaderMap& hea
   switch (result.status_) {
   case LoadDnsCacheEntryStatus::InCache:
     ASSERT(cache_load_handle_ == nullptr);
-    ENVOY_STREAM_LOG(debug, "DNS cache entry already loaded, continuing", *decoder_callbacks_);
+    ENVOY_STREAM_LOG_EVENT(debug, "DNS cache entry already loaded, continuing",
```

Comment:
this event name looks off - no? It does not follow the convention set by other event names and there is no event message?